src/pages/home/report/ReportActionItem.js

```diff
@@ -70,59 +67,45 @@ const propTypes = {
 
 const defaultProps = {
     draftMessage: '',
-    hasOutstandingIOU: false,
 };
 
-class ReportActionItem extends Component {
-    constructor(props) {
-        super(props);
-        this.popoverAnchor = undefined;
-        this.state = {
-            isContextMenuActive: ReportActionContextMenu.isActiveReportAction(props.action.reportActionID),
-        };
-        this.checkIfContextMenuActive = this.checkIfContextMenuActive.bind(this);
-        this.showPopover = this.showPopover.bind(this);
-        this.renderItemContent = this.renderItemContent.bind(this);
-        this.toggleReaction = this.toggleReaction.bind(this);
-    }
+function ReportActionItem(props) {
```

Comment:
Do we wanna go with `function` or arrow syntax for these component definitions?

Reply:
I am on team `function`

Reply:
<img width="488" alt="Screenshot 2023-03-31 at 4 26 35 PM" src="https://user-images.githubusercontent.com/31285285/229177066-11d555a7-752c-49f7-bb26-ef0097f0f2a5.png">

It seems we have an existing linter rule that doesn't let us pass functions that use `function` as props. So let's stick with the arrow syntax?